src/components/Asset/AssetActions/Pool/Remove/index.module.css

```diff
@@ -50,25 +50,30 @@
 }
 
 .output {
-  composes: output from '../Add/Output.module.css';
+  border-top: 1px solid var(--border-color);
+  padding: var(--spacer);
+  padding-bottom: calc(var(--spacer) / 1.5);
+  display: grid;
+  gap: var(--spacer);
+  grid-template-columns: 1fr 1fr;
+  margin-left: -2rem;
```

Comment:
there it is, dealing with `margin` & `padding` already

```css
composes: container from '../Section/index.module.css';
```

Reply:
This answer is valid for all the css related comments: 
I just copy pasted stuff so it doesn't break and looks reasonable. It's always under the assumption that it will be fixed/made prettier in a later pr 🤣